x/denommetadata/keeper/keeper_test.go

```diff
@@ -0,0 +1,58 @@
+package keeper_test
+
+import (
+	"testing"
+
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/dymensionxyz/dymension/v3/app/apptesting"
+
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+type KeeperTestSuite struct {
+	apptesting.KeeperTestHelper
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(KeeperTestSuite))
+}
+
+func (suite *KeeperTestSuite) SetupTest() {
+	app := apptesting.Setup(suite.T(), false)
+	ctx := app.GetBaseApp().NewContext(false, tmproto.Header{})
+
+	suite.App = app
+	suite.Ctx = ctx
+}
+
+func (suite *KeeperTestSuite) TestCreateDenom() {
+	keeper := suite.App.DenomMetadataKeeper
+
+	err := keeper.CreateDenomMetadata(suite.Ctx, suite.getTestMetadata())
```

Comment:
I would add:
1. validation that the denom metadata was actually created in the bank
2. trying to create the same denom and verify I get an error which is already exists

Reply:
done